code/modules/assembly/helpers.dm

```diff
@@ -28,6 +28,10 @@
 		return 1
 	return 0
 
+/proc/ismonitor(O)

```

Comment:
Its not necessary but all the other assemblies had an is() proc so I thought why not?